src/pages/404.tsx

```diff
@@ -28,3 +29,17 @@ const NotFoundPage: React.FC = () => (
 );
 
 export default NotFoundPage;
+
+export const query = graphql`
+  query ($language: String!) {
```

Comment:
1. All Queries should have a name (this is for all page Queries in the PR). Here it could be `NotFoundPageQuery`
2. If you use the locales Query on multiple places, you can create a GraphQl Fragment to not have to write it again and again

Reply:
The name of the query should also be put in the GraphQL query itself:

```
query NotFoundPageQuery ($language: String!) {
```